.gitignore

```diff
@@ -253,3 +253,4 @@ paket-files/
 
 *.suo
 src/ReactiveDomain/.vs/ReactiveDomain/v14/.suo
+/src/ReactiveDomain/ReactiveDomain/Fakes/ReactiveUI.fakes
```

Comment:
Your csproj file includes this, but you have it excluded from Git. Something's not right here.

Reply:
not needed, I'll remove it